commit ec7b317a91e4e2412f5f3a71d8080df286c4e812 (refs/changes/29/2629/1)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2017-09-20T14:22:36-07:00 (2 years, 1 month ago)
    
    HEAD-2366 occassional tar crash can break COAL build on Mac: "Unable to copy files to mount"

diff --git a/bin/build-usb-image b/bin/build-usb-image
index 933cf259..2ab2ea74 100755
--- a/bin/build-usb-image
+++ b/bin/build-usb-image
@@ -203,7 +203,7 @@ function copy_to_mount
 {
     if [[ $TAR_BUILD_IS_DIRECTORY == 1 ]]; then
         (cd $TAR_BUILD_FILENAME \
-            && ${TAR} -c .[a-zA-Z]* * | ${TAR} -C $MNT_DIR -xovf -) \
+            && rsync -av .[a-zA-Z]* * $MNT_DIR/) \
             || fatal "Unable to copy files to mount"
     else
         (cd ${MNT_DIR} && ${TAR} --no-same-owner \
